packages/wrangler/src/pages/dev.tsx

```diff
@@ -457,6 +457,10 @@ async function spawnProxyProcess({
 	port?: number;
 	command: (string | number)[];
 }): Promise<undefined | number> {
+	if (port !== undefined) {
```

Comment:
This will skip running the command if a port is specified, which isn't the intended behavior. Can you also check to see if the `command` is empty before early exiting?